### PR TITLE
Enable default permissions

### PR DESCRIPTION
**Pull request by yurabakhtin**
Enable default permissions for Content Containers: Space and/or User

Issue: https://github.com/humhub/humhub/issues/4670
Reason described here https://github.com/humhub/humhub/issues/4670#issuecomment-735766589 after first screenshot.

### CODE REVIEW

File: module.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
         "organisation",
         "sharing"
     ],
-    "version": "0.12.1",
+    "version": "0.13.0",
     "humhub": {
-        "minVersion": "1.7"
+        "minVersion": "1.8"
```
Comment (luke-) [inline module.json:14]: Is this change no longer compatible with 1.7?

Comment (yurabakhtin) [reply module.json:14]: ah yes, the change doesn't break 1.7, then as I understood we should keep 1.7 here and in all other similar changes in other modules.

Comment (yurabakhtin) [reply module.json]: ok I reverted min version to 1.7. in all external modules where I did the change today.

Comment (luke-) [reply module.json]: Great! Thanks!

**Review by luke- — APPROVED** [review]

**Pull request merged**